hyperwallet/tests/test_api.py

```diff
@@ -1163,6 +1163,105 @@ def test_deactivate_paper_check_success_with_notes(self, mock_post):
 
     '''
 
+    Transfers
```

Comment:
if you run **make coverage** then you can face some errors like this: 
 '''======================================================================
ERROR: test_failed_connection (hyperwallet.tests.test_client.ApiClientTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/gmeyer/IntellijWorkspace/python-sdk-aseveryn-epam/hyperwallet/tests/test_client.py", line 19, in setUp
    SERVER
TypeError: __init__() takes exactly 5 arguments (4 given)

======================================================================
ERROR: test_receive_non_json_response (hyperwallet.tests.test_client.ApiClientTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/gmeyer/IntellijWorkspace/python-sdk-aseveryn-epam/hyperwallet/tests/test_client.py", line 19, in setUp
    SERVER
TypeError: __init__() takes exactly 5 arguments (4 given)'''

Reply:
In scope of this pull request there were no changes in apiclient constructor, but in pull request "Layer 7 encryption" number of arguments in apiclient constructor had been changed, so it seems you have an intersection of those two pull requests. Please check it again